requirements/common_constraints.txt

```diff
@@ -20,6 +20,3 @@ Django<4.0
 elasticsearch<7.14.0
 
 setuptools<60
-
-# redis 4 client doesn't play nicely with redis 3 server
-redis<4
```

Comment:
Do we know if we're not using a redis 3 server anymore?

Reply:
common_constraints is actually copied out of https://github.com/openedx/edx-lint/blob/master/edx_lint/files/common_constraints.txt by make upgrade, it looks like the redis constraint was recently removed

Reply:
Nice